### ISSUE TITLE
Upload of sales progression notes in real time

### ISSUE THREAD

**Issue by lcantrell79** [reapit/foundations#8714]
Please take a look at our FAQ's before submitting an issue, as your question may already be covered in the documentation https://foundations-documentation.reapit.cloud/faqs

**Summary**
Where are notes best stored in Reapit via the API in real time? Would it be a  journal entry? Is there a way for a Reapit user to view these chronologically somewhere?

**Detail and or supporting links and screenshots**
Looking for a way to pass notes real time to Reapit via the API.

**Comment by AshDeeming** [comment]
Hi @lcantrell79 
Currently journalEntries can be used to post updates, however currently these are two types you can post 
MA - for Property Matches and MI - For Miscellaneous
Additional types can be requested along with a requirement case for review.

**Comment by lcantrell79** [comment]
@AshDeeming Would Sales progression type notes be considered? Where the agent could then filter by this type and view notes specific to the sale progress.

**Comment by AshDeeming** [comment]
Hi @lcantrell79 
I'll convert this to a feature request for review/consideration by the relevant team

**Comment by lcantrell79** [comment]
@plittlewood-rpt Yes, that definitely fits our use case. I assume there's no limit to how many notes are displayed on the right of the Offer and Sales Progress screen? Some agents do add many notes :)

**Comment by plittlewood-rpt** [comment]
No it acts like any other activity feed in AgencyCloud. We'll discuss this here and look at the best way of implementing it. Thanks for coming back to me
